pkg/ccl/backupccl/testdata/backup-restore/alter-schedule/missing-schedule

```diff
@@ -21,24 +21,6 @@ exec-sql
 drop schedule $fullID
 ----
 
-# The schedule is now in an invalid state.
-
-query-sql
-with schedules as (show schedules) select id, state, recurrence, owner, command from schedules where label='datatest' order by command->>'backup_type' asc;
-----
-$incID Waiting for initial backup to complete @daily root {"backup_statement": "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH detached", "backup_type": 1, "chain_protected_timestamp_records": true}
-
-exec-sql expect-error-regex=(pq: incremental backup schedule \d* has no corresponding full backup schedule; drop schedule \d* and recreate)
-alter backup schedule $incID set recurring '@hourly';
-----
-regex matches error
-
-exec-sql
-drop schedule $incID
-----
-
-# Test that dropping the incremental backup is not an issue.
-
```

Comment:
i dont think you meant to delete line 40 here?

Reply:
note: for these little nit comments, just go ahead and resolve the comments one you've addressed them.